content/03.heterogeneity.md

```diff
@@ -1,23 +1,24 @@
 ### Managing disparities in data generation is required for robust analysis
 
-Rare disease data often suffers from artifacts introduced by non-biological phenomena such as batch or assay platform. 
+<!-- TODO: Refs! -->
+Rare disease data often suffers from artifacts introduced by non-biological phenomena such as batch or assay platform.
 The consequences of these artifacts are amplified when there are few samples and heterogeneous phenotypes.
-Furthermore, datasets are often combined from multiple small studies where biological characteristics are confounded by technical variables. 
-Collaboration with domain experts may result in unexpected insight into potential sources of variation. 
-The authors experienced this when studying neurofibromatosis type 1 (NF1). 
-The NF1 datasets were comprised of samples obtained with different surgical techniques, resulting in differences that were a consequence of sample collection method rather than differences in biology.
-Consequently, careful assessment of and accounting for confounding factors is critical to identifying meaningful features within a dataset. 
+Furthermore, datasets are often combined from multiple small studies where biological characteristics are confounded by technical variables.
+Collaboration with domain experts may result in unexpected insight into potential sources of variation.
+As an example, consider a study of neurofibromatosis type 1 (NF1) datasets. <!-- TODO: REF -->
+The NF1 datasets were comprised of samples obtained with different surgical techniques, resulting in differences that were a consequence of sample collection method rather than differences in biology. <!-- TODO: not sure how this works here - can we add a take-home point? -->
```

Comment:
This suggests I cut too much information in a previous edit. This section use to describe the methodology a little more - surgical collection vs laser ablation (i.e. burning) + surgical removal of tumors, and clinical collaborators were concerned that this would introduce technical differences (such as heat shock response...)

Reply:
tracking in #113